test/integration_test.py

```diff
@@ -503,24 +515,29 @@ def _test_other_endpoints(self):
     def _test_manifest(self, catalog: CatalogName):
         for format_, validator, attempts in [
             (None, self._check_manifest, 1),
-            ('compact', self._check_manifest, 1),
-            ('terra.bdbag', self._check_terra_bdbag, 1),
-            ('terra.pfb', self._check_terra_pfb, 1),
-            ('curl', self._check_curl_manifest, 1),
+            (ManifestFormat.compact, self._check_manifest, 1),
+            (ManifestFormat.terra_bdbag, self._check_terra_bdbag, 1),
+            (ManifestFormat.terra_pfb, self._check_terra_pfb, 1),
+            (ManifestFormat.curl, self._check_curl_manifest, 1),
         ]:
-            with self.subTest('manifest',
-                              catalog=catalog,
-                              format=format_,
-                              attempts=attempts):
-                assert attempts > 0
-                args = dict(catalog=catalog)
-                if format_ is not None:
-                    args['format'] = format_
-                for attempt in range(attempts):
-                    start = time.time()
-                    response = self._check_endpoint('/manifest/files', args=args)
-                    log.info('Request %i/%i took %.3fs to execute.', attempt + 1, attempts, time.time() - start)
-                    validator(catalog, response)
+            if format_ in self.metadata_plugin(catalog).manifest_formats:
+                format_ = format_.value
+                with self.subTest('manifest',
+                                  catalog=catalog,
+                                  format=format_,
+                                  attempts=attempts):
+                    assert attempts > 0
+                    args = dict(catalog=catalog)
+                    if format_ is not None:
+                        args['format'] = format_
+                    for attempt in range(attempts):
+                        start = time.time()
+                        response = self._check_endpoint('/manifest/files', args=args)
+                        log.info('Request %i/%i took %.3fs to execute.', attempt + 1, attempts, time.time() - start)
+                        validator(catalog, response)
+            else:
+                log.info('Catalog %r doe snot support manifest format %r; skipping subtest',
```

Comment:
```suggestion
                log.info('Catalog %r does not support manifest format %r; skipping subtest',
```

No nasal dripping deers please ;)